### PR TITLE
docs: Adds info on how the calendar works

### PR DESCRIPTION
**Pull request by johntron**

### CODE REVIEW

File: docs/README.md [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+## Services
+
+![Service diagram](./services.png)
+
+DMS calendar communicates with:
+
+* MariaDB: holds persistent data (e.g. calendar events, attendees, etc.); MySQL-compatible database server
+* DMS Active Directory (LDAP): connects the calendar to DMS accounts used across DMS's other services; account / authentication server
+* Braintree: credit card processesing; third-party service
+* Twilio: sends SMS notifications; third-party service
```
Comment (aceat64) [inline docs/README.md]: Nitpick: maybe swap the Twilio and Sparkpost lines so the order matches the image.

File: docs/README.md [hunk 2]
```diff
@@ -0,0 +1,44 @@
...
+* Sparkpost: sends email notifications? third-party service
+
+
+## Webserver
```
Comment (aceat64) [inline docs/README.md:19]: The webserver is actually just Apache and PHP-FPM. You're correct that `/index.php` is effectively what gets invoked, though all of the important stuff is in `/src`.

**Review by aceat64 — APPROVED** [review]
Tentatively approved. If you'd like to make some updates based on these suggestions, let me know and I'll hold off on merging until you're done.

File: docs/README.md (hunk 1 above)
Comment (johntron) [reply docs/README.md]: Good catch - updated.

File: docs/README.md (hunk 2 above)
Comment (johntron) [reply docs/README.md:19]: The dev server uses HHVM though, right? Maybe I'll update this to describe production and development.

Regarding /index.php, I'll add more detail about what happens after /index.php is invoked.

Comment (aceat64) [reply docs/README.md:19]: I don't think I've ever looked at the dev server, so I can't say for sure.

Comment (johntron) [reply docs/README.md:19]: OK - now that I know we use Apache / PHP-FPM in prod, I'll update the docs accordingly.

**Pull request merged**